plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/DockerExtConfBindingProvider.java

```diff
@@ -55,18 +63,27 @@ public String get() {
             return null;
         }
 
+        if (volumeOptions == null) {
```

Comment:
as @Inject on volumeOptions is not optional, I think that we could get rid of default value from the code as it becomes a mandatory parameter